src/NetworkFunction/README.md

```diff
@@ -41,6 +41,13 @@ directive:
       variant: ^Create$|^CreateViaIdentity$|^CreateViaIdentityExpanded$|^Update$|^UpdateViaIdentity$
     hide: true
   - where:
-      verb: Set
+      subject: (.)*(ByResourceGroup)$
     hide: true
+  - where:
+      subject: (.)*(BySubscription)$
+    hide: true
```

Comment:
You can use `remove` if you don't need these cmdlets.

Reply:
We want to generate them, but not show them. Because, internally, our List cmdlet will be using these functions.